### PR TITLE
add podman remote system df

### PR DESCRIPTION
**Pull request by baude**
Signed-off-by: Brent Baude <bbaude@redhat.com>

### CODE REVIEW

File: pkg/api/server/register_system.go [hunk 1]
```diff
@@ -27,6 +27,22 @@ func (s *APIServer) registerSystemHandlers(r *mux.Router) error {
 	//   500:
 	//     $ref: "#/responses/InternalError"
 	r.Handle(VersionedPath("/libpod/system/prune"), s.APIHandler(libpod.SystemPrune)).Methods(http.MethodPost)
+	// swagger:operation GET /libpod/system/df libpod df
+	// ---
+	// tags:
+	//   - system
+	// summary: Show disk usage
+	// description: Return information about disk usage for images, containers, and volumes
```
Comment (TomSweeneyRedHat) [inline pkg/api/server/register_system.go:35]: nit of a nit, would prefer  "containers, images, and volumes".  OCD mini flair.

File: pkg/api/handlers/libpod/system.go [hunk 2]
```diff
@@ -3,6 +3,8 @@ package libpod
 import (
 	"net/http"
 
+	"github.com/containers/libpod/pkg/domain/infra/abi"
+
```
Comment (rhatdan) [inline pkg/api/handlers/libpod/system.go:7]: Remove space

File: pkg/api/server/register_system.go [hunk 3]
```diff
@@ -27,6 +27,22 @@ func (s *APIServer) registerSystemHandlers(r *mux.Router) error {
...
+	// produces:
+	// - application/json
+	// responses:
+	//   200:
+	//     $ref: '#/responses/SystemDiskUse'
+	//   400:
+	//     $ref: "#/responses/BadParamError"
```
Comment (vrothberg) [inline pkg/api/server/register_system.go:42]: We're not returning 400 at the moment. Shall we ditch it here?

**Comment by baude** [comment]
fixed up

**Comment by TomSweeneyRedHat** [comment]
LGTM

**Pull request merged**